source/core.py

```diff
@@ -286,12 +291,18 @@ def handleScreenOrientationChange(self, lParam):
 			import winUser
 			# Resolution detection comes from an article found at https://msdn.microsoft.com/en-us/library/ms812142.aspx.
 			#The low word is the width and hiword is height.
-			if winUser.LOWORD(lParam) > winUser.HIWORD(lParam):
+			if (winUser.LOWORD(lParam) > winUser.HIWORD(lParam):

```

Comment:
nit: Extraneous outer parens. I wouldn't normally comment except you don't do it anywhere else and it's unnecessarily new in this latest commit. :)